### PR TITLE
Start files for new gifio guide

### PR DESCRIPTION
**Pull request by TheKitty**
FIles for the new gifio guide in the Adafruit Learning System

### CODE REVIEW

File: CircuitPython_gifio/Feather/README.gif
Comment (jepler) [inline CircuitPython_gifio/Feather/README.gif:1]: looks like this should have been `README.md`, not `README.gif`?

**Review by jepler — CHANGES_REQUESTED** [review]
this looks incomplete (no .py code?) did you really intend it to be merged already or it still in progress?

**Review by jepler — APPROVED** [review]
@TheKitty will add files back as needed, giving an approve review so the PR can be merged when green on the CI checks.

**Comment by TheKitty** [comment]
Thanks @jepler

**Pull request merged**